feg/protos/s6a_proxy.proto

```diff
@@ -140,6 +142,10 @@ message UpdateLocationRequest {
     bool skip_subscriber_data = 3; // bit 2
     // Send Cancel Location to other EPCs serving the UE
     bool initial_attach = 4; // bit 5
+    // Send Dual Registration 5G Indicator
+    bool dual_registration_5g_indicator = 5; // bit 8
```

Comment:
isn't it this a bad field?

Reply:
I am sorry I didn't understand , Can you elaborate ?

Reply:
Is `dual_registration_5g_indicator` necessary?
I know we discussed this and I may have forgotten something. But is this field related with the feature of adding a secondary rat `nr_as_secondary_rat`

The protocol talks about a deregistration. Per that description I dont see the relationship with nr_as_secondary_rat

![image](https://user-images.githubusercontent.com/16157139/114222421-6c9bff80-9923-11eb-8355-9ef78fff57bb.png)